tests/plugins/test_time_slider_choropleth.py

```diff
@@ -5,16 +5,19 @@
 """
 
 import json
+import sys
 
 import numpy as np
 import pandas as pd
+import pytest
 from branca.colormap import linear
 
 import folium
 from folium.plugins import TimeSliderChoropleth
 from folium.utilities import normalize
 
 
+@pytest.mark.xfail(sys.version_info[0:2] == (3, 8), reason="too modern for py38")
```

Comment:
We should probably drop Python 3.8 in October at its EOL. This should be fine for now.